packages/provider-elasticsearch/src/index.js

```diff
@@ -93,23 +93,13 @@ let ElasticsearchProvider = (config = { request: {} }) => ({
 
     let metaObj = { request, requestOptions }
 
-    try {
-      // Log Request
-      node._meta.requests.push(metaObj)
-      let count = counter.inc()
-      debug('(%s) Request: %O\nOptions: %O', count, request, requestOptions)
-      let { body } = await search(request, requestOptions)
-      metaObj.response = body
-      debug('(%s) Response: %O', count, body)
-    } catch (e) {
-      console.error({ e })
-      metaObj.response = e
-      node.error = e.meta.body.error
```

Comment:
We don't need this piece of code atm. Here's the context on why it was added: https://github.com/futil-js/contexture-elasticsearch/pull/145/files#r464778174. However it looks like this code was changed after Sam's PR to throw an error again, making his original intention not work anymore.